tests/MoBi.R.Tests/DomainHelperForSpecs.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.UnitSystem;
+
+namespace MoBi.R.Tests
```

Comment:
can't we reuse the stuff from the MoBi.Test file? Just a question. It's already bad enough that we are defining stuff like this. We should only do it once